py3status/modules/arch_updates.py

```diff
@@ -1,120 +1,102 @@
 # -*- coding: utf-8 -*-
-
 """
 Display number of pending updates for Arch Linux.
 
-This will display a count of how many 'pacman' updates are waiting
-to be installed and optionally a count of how many 'aur' updates are
-also waiting.
-
 Configuration parameters:
-    cache_timeout: How often we refresh this module in seconds (default 600)
-    format: Display format to use
-        (default 'UPD: {pacman}' or 'UPD: {pacman}/{aur}')
-    hide_if_zero: Don't show on bar if True
-        (default False)
-    include_aur: Set to True to use 'cower' to check for AUR updates
-        (default False)
+    cache_timeout: refresh interval for this module (default 600)
+    check_aur: check AUR for pending updates (default None)
+    check_pacman: check pacman for pending updates (default None)
+    format: display format for this module
+        *(default '[\?if=pacman&if=!aur PAC: {pacman}][\?if=!pacman&if=aur AUR: {aur}]
+        [\?if=pacman&if=aur UPD: {pacman}/{aur}]')*
 
 Format placeholders:
-    {aur} Number of pending aur updates
-    {pacman} Number of pending pacman updates
-    {total} Total updates pending
+    {aur} number of pending AUR updates
+    {pacman} number of pending pacman updates
+    {total} number of pending total updates
 
 Requires:
-    cower: Needed to display pending 'aur' updates
+    cower: a simple AUR downloader
 
 @author Iain Tatch <iain.tatch@gmail.com>
 @license BSD
 """
 
-import subprocess
-import sys
-
-FORMAT_PACMAN_ONLY = 'UPD: {pacman}'
-FORMAT_PACMAN_AND_AUR = 'UPD: {pacman}/{aur}'
-LINE_SEPARATOR = "\\n" if sys.version_info > (3, 0) else "\n"
+STRING_UNAVAILABLE = 'disabled'
 
 
 class Py3status:
+    """
+    """
     # available configuration parameters
     cache_timeout = 600
-    format = ''
-    hide_if_zero = False
-    include_aur = False
+    check_aur = None
+    check_pacman = None
+    format = '[\?if=pacman&if=!aur PAC: {pacman}][\?if=!pacman&if=aur AUR: {aur}]' + \
+        '[\?if=pacman&if=aur UPD: {pacman}/{aur}]'
```

Comment:
no the format should stay as before to allow `include_aur` legacy also formats should look clean.  If they aren't then we 

users can add whatever format they want but the default is still the default and depends on `include_aur` it is like that for legacy reasons not just for fun.

`\?if=pacman&if=aur` I don't think this will work currently